src/pages/Categories/CategoriesBeta.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+	<www-page class="category-beta">
+		<div class="row">
```

Comment:
Please remove foundation classes: `row` `columns` `small-12` and use tailwinds classes instead